CoEpi/ui/alerts/AlertsViewController.swift

```diff
@@ -7,26 +7,70 @@
 //
 
 import UIKit
+import RxSwift
+import RxCocoa
 
 class AlertsViewController: UIViewController {
+    private let viewModel: AlertsViewModel = .init()
+    private let dataSource: AlertsDataSource = .init()
 
+    @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var contactAlerts: UITableView!
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    private let disposeBag = DisposeBag()
 
-        // Do any additional setup after loading the view.
+    private static func formatTitleLabel(count: Int) -> String {
+        let title: String = "\(count) new contact alert"
+        
+        if count < 2 {
+            return title
+        }
+        return title + "s"
     }
+    
+    override func viewDidLoad() {
+        contactAlerts.register(cellClass: AlertCell.self)
+
+        viewModel.alerts.asObservable()
+            .subscribe(onNext: { [unowned self] alerts in
```

Comment:
Let's avoid `unowned`. It doesn't really bring a benefit and makes the code more error prone (e.g. in refactorings).